mkdocs.yml

```
@@ -116,66 +114,37 @@ nav:
-                  - microservices/device/services/device-usb-camera/walkthrough/deployment.md
-                  - microservices/device/services/device-usb-camera/walkthrough/general-usage.md
-                  - microservices/device/services/device-usb-camera/walkthrough/custom-build.md
-              - Supplementary Information:
-                  - microservices/device/services/device-usb-camera/supplementary-info/USB-protocol.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/discovery.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/advanced-options.md
-          - Device Virtual: microservices/device/services/device-virtual/Ch-VirtualDevice.md
-          - Remote Deployment:
-              - Non-Secure Mode: microservices/device/remote/Ch-RemoteNonSecure.md
-              - Secure Mode: microservices/device/remote/Ch-RemoteSecure.md
+        - Device Service: microservices/device/DeviceService.md
+        - Getting Started: microservices/device/GettingStarted.md
+        - Configuration: microservices/device/Configuration.md
+        - API Reference: microservices/device/ApiReference.md
+        - Device Service SDK:
+            - Purpose: microservices/device/sdk/Purpose.md
+            - Getting Started: 
+                -  C Device SDK:
+                   - GettingStartedSDK-C: microservices/device/sdk/devicesdk-getting-started/GettingStartedSDK-C.md
+                -  Go Device SDK:
+                   - GettingStartedSDK-Go: microservices/device/sdk/devicesdk-getting-started/GettingStartedSDK-Go.md
+            - APIs:
+                - C Device SDK :
+                   - APIs: microservices/device/sdk/api/CDeviceSDK/CDeviceSDKAPI.md
+                   - Dependencies: 
+                      - IOTech IOT Utilities: microservices/device/sdk/api/CDeviceSDK/Dependencies/CUtilities.md
+                - Go Device SDK: 
+                   - APIs: microservices/device/sdk/api/GoDeviceSDK/GoDeviceSDKAPI.md
+            - Source Code: microservices/device/sdk/SourceCode.md
+            - Additional Details:
+                - Custom Configuration: microservices/device/sdk/details/CustomConfiguration.md
+                - Custom Rest Apis: microservices/device/sdk/details/CustomRestApis.md
+                - Custom Service Metrics: microservices/device/sdk/details/CustomServiceMetrics.md
+                - Secrets: microservices/device/sdk/details/Secrets.md
+        - Additional Details: 
+            - Command-line Options: microservices/device/details/CommandLine.md
+            - Environment Variables: microservices/device/details/EnvironmentVariables.md
+            - Device Discovery: microservices/device/details/DeviceDiscovery.md
+            - Seeding Secrets: microservices/device/details/SeedingSecrets.md
+            - Service Metrics: microservices/device/details/ServiceMetrics.md
+        - Services: microservices/device/ApiReference.md
```

Why did you remove all the existing services?
Please restore them.

```suggestion
        - Services: 
```

---

updated